app/controllers/stories_controller.rb

```diff
@@ -369,7 +376,7 @@ def assign_listings
   def redirect_to_lowercase_username
     return unless params[:username] && params[:username]&.match?(/[[:upper:]]/)
 
-    redirect_to "/#{params[:username].downcase}", status: :moved_permanently
+    redirect_to "/#{params[:username].downcase}" + internal_nav_param, status: :moved_permanently
```

Comment:
could we extract this into a helper? or come up with a way to make it easier for a developer to know when they should be adding the `internal_nav_param` or not? right now, from looking at this, i have no idea when i should append this to a path and when i don't need to, so i have a feeling that it wouldn't be immediately clear when you'd need to include it in a redirected path or not

Reply:
Agreed, maybe the helper could have a comment explaining what `i=i` is and when to use it. Thanks!

ps. `"#{url}?i=i"` is more idiomatic and faster than `"#{url}" + "?i=i"

Reply:
Agreed, maybe the helper could have a comment explaining what `i=i` is and when to use it. Thanks!

ps. `"#{url}?i=i"` is more idiomatic and faster than `"#{url}" + "?i=i"

Reply:
Agreed, maybe the helper could have a comment explaining what `i=i` is and when to use it. Thanks!

ps. `"#{url}?i=i"` is more idiomatic and faster than `"#{url}" + "?i=i"